app/objects/videos/takeoff-video.js

```diff
@@ -0,0 +1,13 @@
+import EmberObject from '@ember/object';
+
+/**
+ * Takeoff video for Ember Leaflet.
+ */
+export default EmberObject.extend({
+  init() {
+    this._super(...arguments);
+    this.set('url', 'assets/images/takeoff.webm');
```

Comment:
any specific video encoding required?

Reply:
There isn't much mention of specifics on the docs; I created the one with peek and it seems to run find
I'll test out an mp4 and report back

Reply:
peek records mp4 with h264 encoding, that also displays cleanly
I may do some trial and error on other encodings in the future